skyportal/handlers/api/source.py

```diff
@@ -249,6 +251,22 @@ async def get_source(
         .unique()
         .all()
     )
+
+    followup_requests_data = []
+    for req in followup_requests:
+        req_data = req.to_dict()
```

Comment:
no need for the to_dict() here, that serializes all the columns you don't use too.

Reply:
These requests were already in the source, so I think we want this.

Reply:
Oh right apologies I thought the to_dict ran on on the transaction, my bad!